db/update scripts/1.03.sql

```diff
@@ -0,0 +1,6 @@
+SET SQL_SAFE_UPDATES
```

Comment:
Not a fan of turning off SQL_SAFE_UPDATES. Is there anyway to proceed without having this?

Reply:
Not really since I need to update all the entries where phased out is currently null :/ No way to write it in terms of the course_code